rllib/algorithms/dqn/default_dqn_rl_module.py

```diff
@@ -140,23 +139,23 @@ def get_initial_state(self) -> dict:
             return {}
 
     @override(RLModule)
-    def input_specs_train(self) -> SpecType:
+    def input_specs_train(self):
         return [
             Columns.OBS,
             Columns.ACTIONS,
             Columns.NEXT_OBS,
         ]
 
     @override(RLModule)
-    def output_specs_exploration(self) -> SpecType:
+    def output_specs_exploration(self):
```

Comment:
I think keeping a typehint is good and clean coding style - although it does not need to be a `SpecType`, but at least a `List[str]` or `List[ColumnType]`

Reply:
We are going to get rid of Spec/SpecDict soon as well for simplicity. They are already NOT used anymore by RLlib.